### PR TITLE
Add test case for pkg/kubelet/container/runtime.go

### PR DESCRIPTION
**Pull request by changshuchao**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by changshuchao** [comment]
@wzshiming could u please review this pr , thx!

**Comment by changshuchao** [comment]
@ehashman  could you please review this pr ? thanks!

**Comment by changshuchao** [comment]
@dchen1107 could you please review this pr , thx!

**Comment by changshuchao** [comment]
@klueska could you please review this pr , thx!

File: pkg/kubelet/container/runtime_test.go [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+		ID:   "3333",
+	})
+	assert.Nil(t, actualNoResult)
+}
+
+func TestParsePodFullName(t *testing.T) {
+	fullNameWithoutLine := "name3ns3"
+	testCases := []struct {
+		expectName      string
+		expectNmaespace string
```
Comment (kikisdeliveryservice) [inline pkg/kubelet/container/runtime_test.go:348]: Throughout func fix typo:

```suggestion
		expectNamespace string
```

Comment (changshuchao) [reply pkg/kubelet/container/runtime_test.go:348]: thanks!

File: pkg/kubelet/container/runtime_test.go [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+			},
+		},
+		{
+			data:        dataError,
+			containerID: &ContainerID{},
+		},
+	}
+
+	var err error
+	for _, test := range testCases {
```
[the remark this answers is not in the document]
Comment (changshuchao) [reply pkg/kubelet/container/runtime_test.go:58]: ok

**Comment by dims** [comment]
@endocrimes are you happy with this now?

**Review by endocrimes — APPROVED** [review]
I'd still prefer to see subtests for future investigators, but we're not strict on them so lgtm.

/lgtm

**Pull request closed without merging**